scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -199,6 +199,24 @@ sealed trait Execution[+T] extends java.io.Serializable { self: Product =>
   def zip[U](that: Execution[U]): Execution[(T, U)] =
     Zipped(this, that)
 
+  /**
+   * Add an extra config for current execution.
+   * This changes of config are isolated and affect only current execution.
+   * All changes will disappear for any next operation on execution.
+   */
+  def addConfig(extraConfig: Config): Execution[T] =
```

Comment:
Agree with @isnotinvain that this method is unnecessary (or should be added in a separate PR).